discord/ext/pages/pagination.py

```diff
@@ -569,7 +569,7 @@ async def update(
             self.buttons = {}
             for button in custom_buttons:
                 self.add_button(button)
-        else:
+        elif self.use_default_buttons:
             self.buttons = {}
             self.add_default_buttons()
```

Comment:
```suggestion
        elif self.use_default_buttons:
            self.buttons = {}
            self.add_default_buttons()
        else
            self.buttons = {}
```

wouldn't that make more sense @Dorukyum

Reply:
This is probably a good idea. I will fix it when I get home. I think this may fix instances where default buttons are enabled and then updated to be disabled.

Reply:
@Icebluewolf update?

If `self.buttons = {}` is present in every case, just put it outside the whole `if`.

Reply:
I am away from my PC for the next few days. While testing Lalas's suggested change I found another existing bug within this chunk of code. I think I have come up with a solution that I can commit but I have been unable to thourally test it. 

I am hoping that #2320 will fix a lot of the problems with pages. I believe it is better to just fix this bug for now even if it is not the best way to fix it as it will be replaced by #2320. I can commit the changes if someone else can test otherwise we will have to wait until Monday.